examples/ITK_Example18_MONAI_affine_Elastix_nonlinear.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.            LoadImageD(keys=["fixed_hand", "moving_hand"]),
</pre><p>Could this warning be avoided? By adding image_only=False? Warning: <em>"monai.transforms.io.dictionary LoadImaged.__init__:image_only: Current default value of argument </em><code><em>image_only=False</em></code><em> as been deprecated since version 1.1. It will be changed to</em><code><em>image_only=True</em></code><em> in version 1.3."</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>